### DISCUSSION TITLE
Is `Themed.root` deprecated? (no)

### DISCUSSION THREAD

**Discussion by TKDev7** [system-ui/theme-ui#2177] [Q&A] [answered]
If I am not mistaken, the docs website is using deprecated/removed stuff such as Themed.root and ColorModeProvider. Is this correct? Sorry I am new to theme-ui

**Comment by hasparus** [comment, marked as answer]
`ColorModeProvider` is not deprecated.
`Themed.root` is also not deprecated. The `as` prop on Themed components was deprecated, but the components are totally fine.

**Reply by hasparus** [reply]
Do you think we could improve the migration notes to make this clearer?
